### PR TITLE
flautomat:0.1.0

### PR DESCRIPTION
**Pull request by Kuchenmampfer**
I am submitting
- [x] a new package
- [ ] an update for a package

This package can be used to visualize abstract automata. It relies on the json format that [FLACI](https://flaci.com/autoedit) ([FLACI on github](https://github.com/gitmh/FLACI/tree/main)) saves to. This allows for intuitive automata design with FLACI and then an import into Typst without relying on raster graphics. Therefore the package can be useful for anyone who studies abstract automata and wants to visualize them.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

**Comment by jneug** [comment]
Nice package. I was looking into supporting more of FLACI in `finite` myself. 

One suggestion: I think you can get around the json read problem by first reading the file and then decoding it with `json.decode`. In my tests it worked fine:

```typst
#flautomat(json.decode(read("Automaton_TEIL2_ Wort mit gerader Anzahl an Nullen.json")), scaling_factor: 1)
```

File: packages/preview/flautomat/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+```bash
+cat Automaton_MY_AUTOMAT.json | jq . > MY_AUTOMAT.json
+```
+- Now import this package into your typst file and use it:
+```typst
+#import "@preview/flautomat:0.1.0": flautomat
+
+...
+
+#flautomat(json("MY_AUTOMAT.json"), scaling_factor: 1)
```
Comment (jneug) [inline packages/preview/flautomat/0.1.0/README.md:18]: Should be `scaling-factor`.

Comment (Kuchenmampfer) [reply packages/preview/flautomat/0.1.0/README.md]: Right, thank you

**Comment by Kuchenmampfer** [comment]
Thank you :)

I have to admit that I only got aware of `finite` after I made this package. I wondered if a new package was really necessary or if I should make a PR to `finite` to add FLACI-support. But I think our architecture is too different to easily translate. And I now also think that having one powerful and one simple and easy to use package is a good thing.

I tried your approach with reading first, but for me it did not work :/ So I guess I will leave this part in the README.

**Comment by jneug** [comment]
I had a similar experience with Fletcher. It came out right after I published `finite`. Otherwise I probably would have build it on top of Fletcher. 

I think the packages are different enough and having a choice is always a good thing. 😀

**Comment by elegaanz** [comment]
Thanks!

**Pull request merged**